sky/engine/public/platform/sky_settings.h

```diff
@@ -17,6 +17,7 @@ struct SkySettings {
   bool start_paused = false;
   bool enable_dart_checked_mode = false;
   bool trace_startup = false;
+  bool complete_timeline = false;
```

Comment:
Renamed to "vm-complete-timeline". (BTW, one could argue that that `base` trace events are not "timeline" events, so maybe there was no conflict in the first place)